modules/player_pc_navigaton.py

```diff
@@ -0,0 +1,164 @@
+from modules.context import context
+from modules.game import decode_string
+from modules.memory import read_symbol, unpack_uint16
+from modules.menuing import BaseMenuNavigator, scroll_to_item_in_bag
+from modules.items import Item, get_item_bag, get_item_storage
+from modules.tasks import task_is_active
+
+
+def menu_index() -> int:
+    cursor_info = read_symbol("gTasks", 0x160, 4)
+    scroll_offset = unpack_uint16(cursor_info[0:2])
+    selected_row = unpack_uint16(cursor_info[2:4])
+    return scroll_offset + selected_row
```

Comment:
Thank you for explaining the tasks abstraction but I opted to switch to use gPlayerPCItemPageInfo and gPlayerPcMenuManager. Since they are easier to find the in code for others in the future.